tests/helpers/blueprint-shim.js

```diff
@@ -0,0 +1,60 @@
+var fs = require('fs-extra');
+var path = require('path');
+
+var stringifyAndNormalize = require('../../lib/utilities/stringify-and-normalize');
+var transformer = require('../../blueprints/addon/index');
+
+var processTemplate = require('../../lib/utilities/process-template');
+
+// The blueprint paths for `bower.json` and `package.json` so we're using the right things.
+var paths = {
```

Comment:
I'm setting this up to allow us to run tests outside of the context of the Ember CLI folder. I'm intentionally using root-relative paths where at all possible.